buildscripts/scripts/mpv.sh

```diff
@@ -2,23 +2,28 @@
 
 . ../../include/path.sh
 
+build=_build$ndk_suffix
+
+
+# Meson must clean the build directory or it will build a static library instead of a shared one
+rm -rf $build
```

Comment:
are you sure this is a general issue and not just with leftover files in your test setup?
pretty sure you can move it back info the `if`

Reply:
You're right. The problem was that meson was complaining about an already initialized build directory